setup.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import os
+from setuptools import setup
+
+
+DESCRIPTION = 'Iris sample data'
+NAME = 'iris_sample_data'
+DIR_ROOT = os.path.abspath(os.path.dirname(__file__))
+DIR_PACKAGE = os.path.join(DIR_ROOT, NAME)
+DIR_DATA = os.path.join(DIR_PACKAGE, 'sample_data')
+
+
+def extract_version():
+    version = None
+    fname = os.path.join(DIR_PACKAGE, '__init__.py')
+    with open(fname) as fin:
+        for line in fin:
+            if (line.startswith('__version__')):
+                _, version = line.split('=')
+                version = version.strip()[1:-1]  # Remove quotation.
+                break
+    return version
+
+
+def extract_description():
+    description = DESCRIPTION
+    fname = os.path.join(DIR_ROOT, 'README.rst')
+    if os.path.isfile(fname):
+        with open(fname) as fin:
+            description = fin.read()
+    return description
+
+
+def extract_package_data():
+    package_data = []
+    offset = len(os.path.dirname(DIR_DATA)) + 1
+    for dpath, dnames, fnames in os.walk(DIR_DATA):
+        globs = set()
+        fpath = os.path.join(dpath[offset:])
+        for fname in fnames:
+            _, ext = os.path.splitext(fname)
+            globs.add('*{}'.format(ext))
+        for glob in globs:
+            package_data.append(os.path.join(fpath, glob))
+    return {NAME: package_data}
+
+
+setup_args = dict(
+    name=NAME,
+    version=extract_version(),
+    description=DESCRIPTION,
+    long_description=extract_description(),
+    platforms=['Linux', 'Max OS X', 'Windows'],
+    license='Open Government',
+    url='https://github.com/SciTools/iris-sample-data',
+    packages=[NAME],
+    package_data=extract_package_data(),
+    classifiers=[
+        'License :: OSI Approved :: Open Government License',
```

Comment:
Also, I don't see it [listed on OSI](https://opensource.org/licenses/alphabetical).